cfme/tests/configure/test_region.py

```diff
@@ -28,5 +28,9 @@ def _reset_region_description(description, view):
     view.save.click()
     view.flash.assert_message('Region "{}" was saved'.format(region_description))
     view.redhat_updates.click()
-    assert view.title.text == 'Settings Region "{} [{}]"'.format(
-        region_description, appliance.server.zone.region.number)
+    if appliance.version < "5.10":
+        assert view.title.text == 'Settings Region "{} [{}]"'.format(
```

Comment:
Required:
I think better to go like
```
reg = "Settings Region" if appliance.version < "5.10" else "CFME Region"
expected_title = '{reg} "{des} [{num}]"'.format(
    reg=reg, des=region_description, num=appliance.server.zone.region.number
)
```
and `single` assert statment

Reply:
Thank you @digitronik